packages/contracts-bedrock/contracts/L1/ResourceMetering.sol

```diff
@@ -168,14 +133,21 @@ abstract contract ResourceMetering is Initializable {
         }
     }
 
+    /**
+     * @notice
+     */
+    function resourceConfig() public virtual returns (SystemConfig.ResourceConfig memory);
```

Comment:
I think it's here mainly due to the `ResourceMetering` contract needing access to the `SystemConfig` that exists as a state var within a contract that inherits it. Public state variables can only override functions w/ external visibility, which would mean that we'd need to perform an extra call whenever we wanted to access the `SystemConfig`'s `resourceConfig` function from within `ResourceMetering` if we went that route w/o any other modifications.

Agree that it's a weird addition to the portal, though. Maybe instead of having the `SystemConfig` live in the portal, we place it in `ResourceMetering`? I think eventually we'll need it for other things in the portal, but for now, that might be a way to clean this up a bit.